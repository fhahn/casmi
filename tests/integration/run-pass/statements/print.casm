rule main = {
    print "Hallo Welt"
    print 1
    print true
    print false
    print 0
    print 4711
    print ["hallo", "hallo", "foo"]
    print ["foo", 1, 1 = 1]
    print [["foo", 1], ["bar", 2]]

    program( self ) := undef
}

init main

