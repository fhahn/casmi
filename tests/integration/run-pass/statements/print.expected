Hallo Welt
1
true
false
0
4711
[ "hallo", "hallo", "foo" ]
[ "foo", 1, true ]
[ [ "foo", 1 ], [ "bar", 2 ] ]
1 step later...
